ssi-jwt/src/lib.rs

```diff
@@ -135,7 +142,9 @@ impl From<NumericDate> for DateTime<Utc> {
     fn from(nd: NumericDate) -> Self {
         let (whole_seconds, fractional_nanoseconds) =
             nd.into_whole_seconds_and_fractional_nanoseconds();
-        Utc.timestamp(whole_seconds, fractional_nanoseconds)
+        // `timestamp` (deprecated) was already doing an unwrap
```

Comment:
Is the reason that `timestamp` was doing `.unwrap()` (and that the behavior was persisted) is that it is actually impossible to hit the fail state? If it is possible to hit the error state, is there a ergonomics reason we don't want to bubble it here? Like would changing `from`'s signature to `Result<Self>` be problematic?

Reply:
Short answer: it doesn't change anything, it's just to avoid the deprecated notice.

Other short answer: all methods around `NumericDate` can panic so let's put it to tomorrow.